resources/migrations/001_update_migrations.yaml

```diff
@@ -7364,6 +7356,14 @@ databaseChangeLog:
         - customChange:
             class: "metabase.db.custom_migrations.MigrateStackedAreaBarComboDisplaySettings"
 
+  - changeSet:
+      id: v50.2024-05-27T15:55:22
+      author: calherries
+      comment: Create sample content
+      changes:
+        - customChange:
+            class: "metabase.db.custom_migrations.CreateSampleContent"
```

Comment:
The migration needs to be moved if the content is updated, so that it matches the version that the sample content was updated from

Reply:
While people can figure this out through trial and error or looking up this PR through the blame, it seems like a comment here could be helpful.